contentcuration/contentcuration/static/js/utils/get_cookie.js

```diff
@@ -1,4 +1,4 @@
-var jQuery = require("jquery");
+var jQuery = require("rawJquery");
```

Comment:
Almost went for a `modules` alias, mapping directly to `node_modules`, but thought it was kind of heavy handed. @rtibbles any thoughts on this?